compiler/types.nim

```diff
@@ -593,7 +593,10 @@ proc typeToString(typ: PType, prefer: TPreferedDesc = preferName): string =
     of tyUncheckedArray:
       result = "UncheckedArray[" & typeToString(t[0]) & ']'
     of tySequence:
-      result = "seq[" & typeToString(t[0]) & ']'
```

Comment:
Breaking code change as the string representation of a type is exposed via Nim's macro system.

Reply:
After all, this is a fix for a compiler crash. This code would be simply crash the compiler with a segfault:

```nim
type CyclicSeq = seq[ref CyclicSeq]
```

I will move it into a custom PR.

Reply:
here it is https://github.com/nim-lang/Nim/pull/13716